staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -1703,6 +1703,31 @@ type ServiceAccountTokenProjection struct {
 	Path string `json:"path" protobuf:"bytes,3,opt,name=path"`
 }
 
+// ClusterTrustBundleProjection describes how to select a set of
+// ClusterTrustBundle objects and project their contents into the pod
+// filesystem.
+type ClusterTrustBundleProjection struct {
+	// Select a single ClusterTrustBundle by object name.  Mutually-exclusive
+	// with SignerName and LabelSelector.
+	// +optional
+	Name *string `json:"name,omitempty" protobuf:"bytes,1,rep,name=name"`
+
+	// Select all ClusterTrustBundles that match this signer name.
+	// Mutually-exclusive with Name.
+	// +optional
+	SignerName *string `json:"signerName,omitempty" protobuf:"bytes,2,rep,name=signerName"`
+
+	// Select all ClusterTrustBundles that match this label selector.  Must not
+	// be null or empty if SignerName is provided.  Mutually-exclusive with
```

Comment:
@ahmedtd can you describe why this is required when signer name is set?

Reply:
Update the comment --- a given signer might maintain multiple disjoint sets of trust anchors under its signer name (for example, in a canarying situation).